CHANGELOG.md

```diff
@@ -40,6 +40,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Fixed inconsistent pointers bug https://github.com/foundry-rs/starknet-foundry/issues/659
 - Fixed an issue where `deploy_at` would not trigger the constructors https://github.com/foundry-rs/starknet-foundry/issues/805
 
+### Cast
+
+#### Added
+
+- `show-config` subcommand to display currently used configuration
+
+### Changed
+
+
```

Comment:
Let's update the changelog, so the changes are on top, as a new version has come out

Reply:
done!